GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -638,7 +639,10 @@ private void blameToolStripMenuItem_Click(object sender, EventArgs e)
             }
 
             blameToolStripMenuItem.Checked = !blameToolStripMenuItem.Checked;
-            BlameSelectedFileDiff();
+            ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+            {
+                await ShowSelectedFileBlameAsync();
+            }).FileAndForget();
```

Comment:
❔
```suggestion
            ThreadHelper.JoinableTaskFactory.RunAsync(ShowSelectedFileBlameAsync).FileAndForget();
```

Reply:
Yes, but this will be changed with #9804 again.